tests/projects/test_databricks.py

```diff
@@ -210,3 +192,18 @@ def test_fetch_and_clean_project(tmpdir):
     for fetched_dir in [fetched0, fetched1]:
         with open(os.path.join(fetched_dir, "MLproject")) as handle:
             assert handle.read() == "Hello"
+
+
+def test_get_run_env_vars(tmpdir):
```

Comment:
Could you add a comment on what this is trying to test? Btw, maybe just write the two asserts out, since this seems a bit indirect for what it's actually doing.